packages/@react-spectrum/datepicker/src/DatePickerSegment.tsx

```diff
@@ -60,13 +60,9 @@ function EditableSegment({segment, state}: DatePickerSegmentProps) {
         'is-placeholder': segment.isPlaceholder,
         'is-read-only': !segment.isEditable
       })}
-      style={{
-        ...segmentProps.style,
-        minWidth: segment.maxValue != null ? String(segment.maxValue).length + 'ch' : null
-      }}
+      style={segmentProps.style}
       data-testid={segment.type}>
-      <span aria-hidden="true" className={classNames(styles, 'react-spectrum-DatePicker-placeholder')}>{segment.placeholder}</span>
-      {segment.isPlaceholder ? '' : segment.text}
+      {segment.isPlaceholder ? <span aria-hidden="true" className={classNames(styles, 'react-spectrum-DatePicker-placeholder')}>{segment.placeholder}</span> : segment.text}
```

Comment:
I'm not seeing this span in the DOM at all, not sure why. It looks like the placeholder text (e.g. mm) is directly inside the div. 🤔

Reply:
Which build are you testing? When I look at bc8e7a2a8d6b89da4f821cefe50488b494433f35 I see the following.
<img width="562" alt="image" src="https://github.com/adobe/react-spectrum/assets/309540/a5d7aaa5-a4d9-43e3-b140-81c915732626">

Reply:
hmm weird, it is working now... not sure what happened before.

Should we get rid of the italics though while we're at it? @dannify seemed to think so